app/readiness.py

```diff
@@ -6,17 +6,24 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-class ReadinessFile:
+class Readiness:
     def __init__(self, readiness_file: Path):
         self.readiness_file = readiness_file
 
     def __enter__(self):
-        logger.debug('Creating readiness file', readiness_file_path=str(self.readiness_file))
-        open(self.readiness_file, 'w').close()
+        self.show_ready()
+        return self
 
     def __exit__(self, *args):
+        self.show_unready()
+
+    def show_ready(self):
+        logger.debug('Creating readiness file', readiness_file_path=str(self.readiness_file))
+        self.readiness_file.touch()
+
+    def show_unready(self):
         logger.debug('Removing readiness file', readiness_file_path=str(self.readiness_file))
         try:
             self.readiness_file.unlink()
         except FileNotFoundError:
-            logger.error('Readiness file not found on exit', readiness_file_path=str(self.readiness_file))
+            logger.debug('Readiness file not found on exit', health_file_path=str(self.readiness_file))
```

Comment:
As `show_unready` might also be called directly:

```suggestion
            logger.debug('Readiness file not found', health_file_path=str(self.readiness_file))
```